Dockerfile

```diff
@@ -88,4 +88,14 @@ RUN rm /var/cache/apk/* \
 COPY config /root/.kube/config
 COPY settings.xml /root/.m2/settings.xml
 
-CMD bash
\ No newline at end of file
+CMD bash
+
+FROM node:alpine
+
+RUN npm install -g redoc-cli
+
+WORKDIR /data
+EXPOSE 8080
```

Comment:
why

Reply:
It's can be started as server to process incoming requests to this port

Reply:
I removed it too. i think we will use just html bundle